### ISSUE TITLE
[BUG] Arrows appear when scrolling on the Y-axis

### ISSUE THREAD

**Issue by kinnn2** [asmyshlyaev177/react-horizontal-scrolling-menu#284]
![3-a4e67fd416](https://github.com/user-attachments/assets/8e9e85b9-3ec2-46bf-98f6-0385c33142d6)

I tested this in an example storybook with random spaces below (see attached image)

When scrolling down, left and right arrows appear at the breakpoint location

I'm not sure if this is the intended design, but in the product I'm creating, the arrows that are created when scrolling down and back up sometimes don't disappear.

I just wanted to say thank you for a great library and hope this is a fixable issue. 

Please understand that sentences written in a translator can be awkward.

**Comment by asmyshlyaev177** [comment]
Hi, it is solvable.

Take a look into how `OnScreenContext` in [Simple](https://asmyshlyaev177.github.io/react-horizontal-scrolling-menu/?path=/story/examples-simple--simple) example used.
```
  const isOnScreen = React.useContext(OnScreenContext);
  const [disabled, setDisabled] = React.useState(isLastItemVisible);
  React.useEffect(() => {
    if (isOnScreen) {
      setDisabled(isLastItemVisible);
    }
  }, [isOnScreen, isLastItemVisible]);
```

**Comment by kinnn2** [comment]
@asmyshlyaev177 
Thank you for your response.
I solved it with the code you mentioned. 
Maybe there is room for this to be improved as a default feature in the future?
It would be nice if there was a way to disable the Y-axis option.

**Comment by asmyshlyaev177** [comment]
> @asmyshlyaev177 Thank you for your response. I solved it with the code you mentioned. Maybe there is room for this to be improved as a default feature in the future? It would be nice if there was a way to disable the Y-axis option.

I will think about it, since you aren't the first who stumble into it. But a bunch of storybook examples should cover most common use cases.

**Comment by asmyshlyaev177** [comment]
Released version `7.1.2` with this fix.
